scripts/reexport_contract_model.py

```
@@ -0,0 +1,281 @@
+        [sys.executable, "-c", probe_code, str(model_path), LEGACY_WARNING_TOKEN],
+        check=True,
+        capture_output=True,
+        text=True,
+    )
+    return json.loads(result.stdout.strip() or "[]")
+
+
+def main(argv: Sequence[str]) -> int:
+    args = parse_args(argv)
+    if args.source_tag == args.target_tag:
+        raise ValueError("--source-tag and --target-tag must differ")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)